LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright 2010, Google Inc. All rights reserved.
-Copyright OpenRefine contributors. All rights reserved.
+Copyright 2012, OpenRefine contributors.
```

Comment:
```suggestion
Copyright 2010, OpenRefine contributors.
```

As I mentioned earlier, I started contributing in 2010. 9c6097a06560fabc8a9ba2c45a23522911cb425f